### ISSUE TITLE
feat: allow serialization of `datetime` that have no timezone

### ISSUE THREAD

**Issue by jkowalleck** [madpah/serializable#74]
Current implementation for serialization helpers fails to render properly, in case a `datetime` has no timezone. Per [JSON schema spec](https://json-schema.org/understanding-json-schema/reference/string#dates-and-times), the "time-offset" is [mandatory](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) to a "date-time".

https://github.com/madpah/serializable/blob/32fbe2a5b2cbfa06ca8899d6e319a120971ee8e1/serializable/helpers.py#L182

from the docs of `datetime.isoformat()`: https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat

> If [utcoffset()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcoffset) does not return None, a string is appended, giving the UTC offset:

in other words: 

> If self.tzinfo is not None, the UTC offset is also attached, _giving giving [sic]_ a full format of 'YYYY-MM-DD HH:MM:SS.mmmmmm+HH:MM'.

----

possible solution(s): (to be discussed)
- assume the timezone of the current system
- assume Zulu/UTC/`+00:00`
